aim/api/status.py

```diff
@@ -58,7 +58,7 @@ class AciStatus(resource.ResourceBase, OperationalResource):
     def __init__(self, **kwargs):
         super(AciStatus, self).__init__({'resource_type': None,
                                          'resource_id': None,
-                                         'sync_status': self.SYNC_PENDING,
+                                         'sync_status': None,
```

Comment:
By doing this, creating a new object can trigger the neighborhood sync propagation. If this was already in sync pending, the propagation would stop at the current object. Keep in mind that we set an object status to SYNC_PENDING right after its creation, so we will never show an empty state to the user